From 0fc79625e55b09ae62a82f3c6501005c690b5fc6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?M=C3=A5rten=20Kongstad?= <marten.kongstad@sonymobile.com>
Date: Mon, 27 Jul 2015 12:24:40 +0200
Subject: [PATCH] OMS-N: adb shell command to access OverlayManagerService

Add a command to communicate with the OverlayManagerService for
debugging purposes. This mirrors the am and pm commands.

This commit restores the functionality after the Nougat rebase from
Sony.

Example use:
    $ adb shell om list
    com.android.systemui
        [ ] com.test.awesome-home-button

    $ adb shell om enable com.test.awesome-home-button

    $ adb shell om list
    com.android.systemui
        [x] com.test.awesome-home-button

Co-authored-by: Martin Wallgren <martin.wallgren@sonymobile.com>
Signed-off-by: Zoran Jovanovic <zoran.jovanovic@sonymobile.com>
Change-Id: If424b8ef6052e4121902b630279c0ebaf416203c
---
 target/product/base.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/product/base.mk b/target/product/base.mk
index 4d70664..5b673ff 100644
--- a/target/product/base.mk
+++ b/target/product/base.mk
@@ -108,6 +108,7 @@ PRODUCT_PACKAGES += \
     mtpd \
     ndc \
     netd \
+    om \
     ping \
     ping6 \
     platform.xml \
-- 
2.9.3

